chapter_6/source_code/fingerprint/fingerprint_process/fingerprint_process.cpp

```diff
@@ -69,11 +69,11 @@ void thinning(Mat& im)
     im *= 255;
 }
 
-int main( int argc, const char** argv )
+int main()
 {
     // Read in an input image - directly in grayscale CV_8UC1
     // This will be our test fingerprint
-    Mat input = imread("/data/fingerprints/101_1.tif", IMREAD_GRAYSCALE);
+    Mat input = imread("data/fingerprints/101_1.tif", IMREAD_GRAYSCALE);
```

Comment:
Could you change path of this to `../../../datasets/DB1_B/101_1.tif` so that it actually makes sense. `/data` was indeed a hardcoded folder on my system, but in general the path I suggest points to the correct file in location to the repository when the executable is created inside the `fingerprint_process` folder.